docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -6,35 +6,47 @@ This section gives a high level overview of the different components that are in
 
 Three main sub-systems are required to facilitate and drive the auction.
 
-### The Orderbook
+### OrderBook
```

Comment:
Do we really want to call it OrderBook camel cased?

Reply:
Tracking all these "standardisation" (sic, US v UK English :laughing:) in https://github.com/cowprotocol/docs-v2/issues/225 which will happen all at the end, but just keeping some level of cohesion til then :smiley: